docs/guides/webhooks.md

```diff
@@ -79,6 +82,24 @@ app.post('/webhooks', express.text({type: '*/*'}), async (req, res) => {
 });
 ```
 
+## Cloudflare workers
+
+// Register webhooks after OAuth completes
+
+```ts
+async function handleFetch(request: Request, env: unknown): Promise<Response> {
+  try {
+    await shopify.webhooks.process({
+      context: {env}, // is object or undefined
+      rawBody: await request.text(), // is a string
+      rawRequest: request,
+    });
+  } catch (error) {
+    console.log(error.message);
+  }
+}
```

Comment:
Should we also consider the `context` argument here, and pass it into `process` as well? Maybe the type of `context` could be an object like

```suggestion
async function handleFetch(request: Request, env: unknown, context: unknown): Promise<Response> {
  try {
    await shopify.webhooks.process({
      context: {env, context}, // is object or undefined
      rawBody: await request.text(), // is a string
      rawRequest: request,
    });
  } catch (error) {
    console.log(error.message);
  }
}
```

though the name `context` here might not make a lot of sense any more? WDYT?